content/faq/security/general.md

```diff
@@ -50,39 +50,39 @@ Docker provides various types of audit logs and log retention varies. For exampl
 
 ### How do we de-provision access to CLI users who use personal access tokens instead of our IdP? We use SSO but not SCIM. 
 
-If SCIM is not enabled, you have to manually remove PAT users from the organization in our system. When SCIM is used this is automated.
+If SCIM isn't enabled, you have to manually remove PAT users from the organization in our system. Using SCIM automates this.
 
 ### What metadata is collected from container images that Scout analyzes?
 
-For information about the metadata stored by Docker Scout, [Data handling](../../scout/data-handling.md).
+For information about the metadata stored by Docker Scout, see [Data handling](../../scout/data-handling.md).
 
-### To which portions of the host filesystem do containers have read and write access? Can containers running as root gain access to admin owned files or directories on the host? 
+### To which portions of the host filesystem do containers have read and write access? Can containers running as root gain access to admin-owned files or directories on the host? 
 
 File sharing (bind mount from the host filesystem) uses a user-space crafted file server (running in `com.docker.backend` as the user running Docker Desktop), so containers can’t gain any access that the user on the host doesn’t already have.
 
 ### How are Extensions within the Marketplace vetting for security prior to placement? 
 
-Security vetting for extensions is on our roadmap however this vetting is not currently done. 
+Security vetting for extensions is on our roadmap however this vetting isn't currently done. 
 
-At present in the marketplace, there are two types of extensions - reviewed and self-published. Reviewed extensions are used and reviewed against a set of criteria, and if they pass they are included in the marketplace with a **Reviewed** label. Self-published extensions are automatically placed in the marketplace with a **Not reviewed** label. 
+At present in the marketplace, there are two types of extensions: reviewed and self-published. Reviewed extensions are used and reviewed against a set of criteria, and if they pass, they're included in the marketplace with a **Reviewed** label. Self-published extensions are automatically placed in the marketplace with a **Not reviewed** label. 
 
-Note that even if an extension is reviewed, it is only reviewed on the first publish. Any updates afterwards are not reviewed. Extensions are not covered as part of Docker’s Third-Party Risk Management Program.
+Note that even if an extension is reviewed, it's only reviewed on the first publish. Any updates afterwards aren't reviewed. Extensions aren't covered as part of Docker’s Third-Party Risk Management Program.
 
 ### Can I disable private repos in my organization via a setting to make sure nobody is pushing images into Docker Hub? 
 
-Currently this is not possible. 
+Currently this isn't possible. 
```

Comment:
I guess this should be a "No" -> https://docs.docker.com/contribute/style/recommended-words/#currently

Reply:
Lol pretty much. Editing.